src/Symfony/Component/DependencyInjection/Container.php

```diff
@@ -290,6 +290,14 @@ public function reset(): void
         $this->envCache = $this->services = $this->factories = $this->privates = [];
     }
 
+    /**
+     * @internal
+     */
+    public function resetEnvCache(): void
+    {
+        $this->envCache = [];
```

Comment:
I don't know exactly, but it looks like it could be reset in `Container::reset()`, so we can avoid adding this method.

Reply:
It is already reset in `Container::reset()`, however `service_container` isn't part of ServicesResetter (and never called in messenger workers)